test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java

```diff
@@ -1451,6 +1457,88 @@ public void itShouldMaximizeAndEquipSelectedWeapon() {
+            withNoEffects(),
+            withProperty("umbrellaState", "broken"),
+            withEquipped(Slot.HAT, "Apriling band helmet"),
+            withUnequipped(Slot.WEAPON),
+            withUnequipped(Slot.OFFHAND),
+            withEquipped(Slot.SHIRT, "Jurassic Parka"),
+            withProperty("parkaMode", "kachungasaur"),
+            withEquipped(Slot.PANTS, "designer sweapants"),
+            withEquipped(Slot.CONTAINER, "bat wings"),
+            withEquipped(Slot.ACCESSORY1, "Cincho de Mayo"),
+            withEquipped(Slot.ACCESSORY2, "combat lover's locket"),
+            withEquipped(Slot.ACCESSORY3, "spring shoes"),
+            withEquippableItem("seal-skull helmet"), // 2283
+            withEquippableItem("ravioli hat"),
+            withEquippableItem("Hollandaise helmet"),
+            withEquippableItem("disco mask"),
+            withEquippableItem("mariachi hat"),
+            withEquippableItem("helmet turtle"),
+            withEquippableItem("coconut shell"),
+            withEquippableItem("Apriling band helmet"),
+            withEquippableItem("toy accordion"),
+            withEquippableItem("hobo code binder"),
+            withEquippableItem("stuffed spooky gravy fairy "),
+            withEquippableItem("stuffed astral badger"),
+            withEquippableItem("magical ice cubes"),
+            withEquippableItem("Roman Candelabra"),
+            withEquippableItem("unbreakable umbrella (broken)"),
+            withEquippableItem("august scepter"),
+            withEquippableItem("bat wings"),
+            withEquippableItem("Jurassic Parka (kachungasaur mode)"),
+            withEquippableItem("old sweatpants"),
+            withEquippableItem("tearaway pants"),
+            withEquippableItem("designer sweatpants"),
+            withEquippableItem("Everfull Dart Holster"),
+            withEquippableItem("cursed monkey's paw"),
+            withEquippableItem("astral mask"),
+            withEquippableItem("Cincho de Mayo"),
+            withEquippableItem("combat lover's locket"),
+            withEquippableItem("spring shoes"),
+            withEquippableItem("turtle totem"),
+            withEquippableItem("pasta spoon"),
+            withEquippableItem("saucepan"),
+            withEquippableItem("disco ball"),
+            withEquippableItem("little paper umbrella"),
+            withEquippableItem("candy cane sword cane"),
+            withEquippableItem("June cleaver"), // 10920
+            withEquippableItem("seal-clubbing club")); // 1
+    String out;
+    String cmd = "maximize(\"" + maxStr + "\", false)";
+    try (cleanups) {
+      out = execute(cmd);
+    }
+    assertFalse(out.isEmpty());
+    System.out.println(out);
+    assertTrue(out.contains("Wielding June cleaver"));
+    assertTrue(out.contains("Putting on designer sweatpants..."));
+    assertContinueState();
+    var requests = getRequests();
+    assertFalse(requests.isEmpty());
+    for (var req : requests) {
```

Comment:
An earlier version used filtered streams to get the first instance of a request that equipped the item.  However the current code generates about 50 requests and 45 are get requests.  The stream used getPostRequestBody which fails the test if it is passed a get request.  So this works around that.  There are almost certainly alternative implementations.